src/fe-text/irssi.c

```diff
@@ -352,6 +353,7 @@ int main(int argc, char **argv)
 	   you have to call setlocale(LC_ALL, "") */
 	setlocale(LC_ALL, "");
 
+	loglev = g_log_set_always_fatal(G_LOG_FATAL_MASK | G_LOG_LEVEL_CRITICAL);
```

Comment:
Ooooh, that's awesome. Right, I didn't realize that the return value of g_log_set_always_fatal is the old mask. Okay, I like this.